TypeRaceAir/race.py

```diff
@@ -0,0 +1,63 @@
+import typing
```

Comment:
Please add a docstring that describes the module (file)

Reply:
like a "readme" doc about the module?


> Please add a docstring that describes the module (file)